polygeist/tools/cgeist/Lib/utils.cc

```diff
@@ -73,33 +73,36 @@ NamespaceKind getNamespaceKind(const clang::DeclContext *DC) {
   return NamespaceKind::Other;
 }
 
-FunctionContext getInputContext(const OpBuilder &B) {
+InsertionContext getInputContext(const OpBuilder &B) {
```

Comment:
All references to this function are gone, except for one use that was added in an asset. Can we remove this function ?

Reply:
Was thinking that it can still be useful for non `MLIRScanner` callers, but it is easy enough to add back when there is a use case, so deleted.